apps/web/modules/settings/organizations/[id]/about-view.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { AboutOrganizationForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Meta, WizardLayout } from "@calcom/ui";
+
+const AboutOrganizationPage = () => {
+  const { t } = useLocale();
+  return (
+    <>
+      <Meta title={t("about_your_organization")} description={t("about_your_organization_description")} />
```

Comment:
For new app routes we don't to use `<Meta />` since it used to set the `MetaContext` state that was used for the page headers. We can now pass that data server side.

We can probably keep the `<Meta />` component in the pages routes.

Reply:
good point! fixed!